test/actions/actions.mocha.coffee

```diff
@@ -46,11 +46,17 @@ describe "Actions", ->
         direction: 'ascending'
       }
       {
+        action: 'fallback',
+        expression: { op: 'ref', name: 'myVar2' }
+      }
+      {
         action: 'limit'
         limit: 10
       }
       { action: 'count' }
       { action: 'sum', expression: { op: 'ref', name: 'myVar' } }
+      { action: 'power', expression: { op: 'ref', name: 'myVar' } }
+      { action: 'absolute', expression: { op: 'ref', name: 'myVar' } }
```

Comment:
absolute should not have an expression... maybe we should add a check to expresionless actions to error out if you give them an expression. ... something for next time :-p